### DISCUSSION TITLE
CascaderNode colour and/or icon for colour picker?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [ant-design-blazor/ant-design-blazor#3205] [Q&A]
Is there a way to render a `CascaderNode` with a colour block or icon?

![image](https://user-images.githubusercontent.com/21806535/228385094-579f10b5-85ac-48bf-9737-5850a68b8db2.png)

**Comment by ElderJames** [comment]
We may add a ItemTemplate for that.
